src/api/instances.rs

```diff
@@ -22,6 +22,7 @@ pub fn fetch_instances() -> std::result::Result<Vec<Instance>, reqwest::Error> {
             .linked
             .iter()
             .filter(|instance| instance.software == Some("lemmy".to_owned()))
+            .filter(|instance| instance.domain.clone().contains(&query_filter.to_lowercase()))
```

Comment:
The query is being lowercased on each iteration now. It would be better for the performance if we save it as lowercased to a variable before the we iterate over all instances.

Reply:
Fixed: ee859e01c5ecff2e8f0a60e19f80c56a2caf92cd